synapse/lib/stormtypes.py

```diff
@@ -7080,6 +7086,15 @@ async def addNode(self, form, valu, props=None):
         else:
             await view.addNode(form, valu, props=props, user=self.runt.user)
 
+    async def detach(self):
+
+        view = self._reqView()
+        if not self.runt.isAdmin(gateiden=view.iden):
+            mesg = 'You must be an admin of the view to detach.'
+            raise s_exc.AuthDeny(mesg=mesg)
```

Comment:
```suggestion
            raise s_exc.AuthDeny(mesg=mesg, user=self.runt.user.iden, username=self.runt.user.name)
```
`user` and `username` are standard in the AuthDeny exception